test/Makefile

```diff
@@ -10,7 +10,7 @@ default: all
 
 # set hard and soft limits for spawn fd-exhaustion test
 ifeq ($(OS),WINNT)
-ULIMIT_TEST=
+ULIMIT_TEST=true
```

Comment:
This isn't cygwin specific, this just wasn't tested correctly. I have a PR that fixes it.